index/slashing/slashing.go

```diff
@@ -2,23 +2,22 @@ package slashing
 
```

Comment:
Summary:
- Metrics
- Refactor/improvements, renames, etc

Most _Consensus_ name referencing was deleted considering yesterday's talk with FC guys. The `slashedAt` is used both for consensus faults and deal faults, so it can't be distinguished. So now its just named _History_.